### PR TITLE
Ensured that Column Metadata window opens with the default selected dataset in the Data View

### LINKED ISSUE #7477: Column Metadata doesn't open with the default selected dataset in the Data View
I have loaded the survey for `rwanda.RDS` file which comes with two datasets `rwanda_daily` and `rwanda_stations`, the default selected data in the `Data View window` is `rwanda_daily` but I select `rwanda_stations` then when I open `Column Metadata` it still open `rwanda_daily` by default rather than `rwanda_stations`. This was abit confusing for user during the workshop and always we were supposed to remind them to change the `dataset`.
image

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #7477 
@rdstern I have made the improvement to the column meta data to change to the current data frame when the current data frame has changed from data view. Please have a look. Thanks
@africanmathsinitiative/developers this is ready for review.
@derekagorhom please have a look.

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -1505,6 +1505,10 @@ Public Class frmMain
         ucrColumnMeta.SetCurrentDataFrame(iIndex)
     End Sub
 
+    Public Sub UpCurrentColumnMetaDataFrame(strDataName As String)
+        ucrColumnMeta.SetCurrentDataFrame(strDataName)
+    End Sub
+
```
Comment (lloyddewit) [inline instat/frmMain.vb:1511]: Remove (see comment below)?
```suggestion
```

File: instat/ucrDataView.vb [hunk 2]
```diff
@@ -230,6 +230,7 @@ Public Class ucrDataView
     End Sub
 
     Public Sub CurrentWorksheetChanged()
+        frmMain.UpCurrentColumnMetaDataFrame(_grid.CurrentWorksheet.Name)
```
Comment (lloyddewit) [inline instat/ucrDataView.vb:233]: `frmMain.ucrColumnMeta` is public. Would it be simpler just to call like this?
```suggestion
        frmMain.ucrColumnMeta.SetCurrentDataFrame(_grid.CurrentWorksheet.Name)
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

File: instat/ucrDataView.vb (hunk 2 above)
Comment (N-thony) [reply instat/ucrDataView.vb]: @lloyddewit this is resolved, thanks.

**Review by lloyddewit — APPROVED** [review]
@N-thony Thanks, looks good

**Comment by lloyddewit** [comment]
@rdstern if you can test/approve, then we can merge, thanks

**Review by rdstern — APPROVED** [review]
@N-thony great.  I am also very happy that these are changes proposed by other than me.  Also they are relatively small, but add to the convenience, and that is important for R-Instat.  Our catch-phrase is "making it easy" and this sort of thing helps!

**Comment by N-thony** [comment]
@lloyddewit I have tried to test this with Linux version unfornately I was getting the following bug, I have fixed it in PR #7537
image

**Pull request merged**